packages/mongodb_atlas/data_stream/hardware/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,111 @@
+---
+description: Pipeline for processing MongoDB Atlas hardware metrics.
+processors:
+  - set:
+      field: ecs.version
+      value: 8.11.0
+  - set:
+      field: event.kind
+      value: event
+  - set:
+      field: event.module
+      value: mongodb_atlas
+  - set:
+      field: event.category
+      value: ["driver"]
```

```suggestion
      value: ["driver"]
```

shouldn't this category be "database"?

https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-category.html#ecs-event-category-database

accordingly find the correct event.type too.